src/theme.ts

```diff
@@ -311,6 +311,30 @@ export const COLORS: {[name: string]: Theme} = {
     DEFAULT_RANK_SHADOW: "#27c5ff",
     DEFAULT_RANK_TEXT: "#ffc20e",
   },
+  radical: {
+    BACKGROUND: "#141321",
+    ICON_CIRCLE: "#FFF",
+    TITLE: "#fe428e",
+    TEXT: "#a9fef7",
+    LAUREL: "#50fa7b",
+    SECRET_RANK_1: "#ff5555",
+    SECRET_RANK_2: "#ff15d9",
+    SECRET_RANK_3: "#973db7",
+    SECRET_RANK_TEXT: "#ff61c6",
+    NEXT_RANK_BAR: "#191326",
+    S_RANK_BASE: "#ffce32",
+    S_RANK_SHADOW: "#ffce32",
+    S_RANK_TEXT: "#f8d847",
+    A_RANK_BASE: "#06b86c",
+    A_RANK_SHADOW: "#06b86c",
+    A_RANK_TEXT: "#f8d847",
+    B_RANK_BASE: "#00006c",
+    B_RANK_SHADOW: "#00006c",
+    B_RANK_TEXT: "#f8d847",
+    DEFAULT_BASE: "#8be9fd",
```

Comment:
You made a mistake about the field name.
DEFAULT_RANK_XXX is required. Not DEFAULT_XXX.

Reply:
ah you're right

Reply:
i've fixed it

Reply:
how about now